lib/Raven/Client.php

```diff
@@ -195,6 +200,7 @@ public function __construct($options_or_dsn = null, $options = array())
         $this->context = new Raven_Context();
         $this->breadcrumbs = new Raven_Breadcrumbs();
         $this->_shutdown_function_has_been_set = false;
+        $this->useCompression = function_exists('gzcompress') && extension_loaded('zip');
```

Comment:
Why does zip extension matter here? Zip extension doesn't provide gzcompress().

Reply:
In older PHP versions (5.4 to 5.6, see [this build](https://travis-ci.org/getsentry/sentry-php/jobs/356383426) of 68eb97c, which is the commit before ce2b5b5 which introduced this fix) the zip extension is not always present... I tried to not check the extension but it wasn't working.